modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -1520,25 +1520,20 @@ export abstract class AbstractEthLikeNewCoins extends AbstractEthLikeCoin {
       }
     }
 
-    const gasLimit = new optionalDeps.ethUtil.BN(this.setGasLimit(params.gasLimit));
+    // Use default gasLimit for cold and custody wallets
+    let gasLimit =
```

Comment:
actually is gasLimit mandatory for tx building if not we can just fetch it at the WP

Reply:
yes, it is mandatory to pass gasLimit... If not, it will throw an error `Error: Cannot convert string to buffer. toBuffer only supports 0x-prefixed hex strings and this string was given: 0xNaN`